src/components/GameView/GameMenu.vue

```diff
@@ -8,19 +8,28 @@
         </v-btn>
       </template>
       <!-- Menu -->
-      <v-list id="game-menu">
-        <v-list-item data-cy="rules-open" @click.stop="showRulesDialog = true">
-          Rules
-          <rules-dialog :hideActivator="true" :show="showRulesDialog" @close="showRulesDialog = false" />
+      <v-list id="game-menu" class="bg-surface-2">
+        <v-list-item data-cy="rules-open">
+          <v-btn class="menu-button" color="surface-2" :size="buttonSize" @click.stop="openRulesDialog">
+            Rules
+          </v-btn>
         </v-list-item>
         <!-- Concede Dialog (Initiate + Confirm) -->
-        <v-list-item data-cy="concede-initiate" @click.stop="openConcedeDialog">Concede</v-list-item>
-        <v-list-item data-cy="stalemate-initiate" @click.stop="openStalemateDialog">
-          Request Stalemate
+        <v-list-item data-cy="concede-initiate">
+          <v-btn class="menu-button" color="surface-2" :size="buttonSize" @click.stop="openConcedeDialog">
+            Concede
+          </v-btn>
+        </v-list-item>
+        <v-list-item data-cy="stalemate-initiate">
+          <v-btn class="menu-button" color="surface-2" :size="buttonSize" @click.stop="openStalemateDialog">
+            Request Stalemate
+          </v-btn>
```

Comment:
Instead of nesting `v-btn` elements inside the list items, we can just use the list item itself and give each list item a different`value` prop. That'll give it the hover and click-ripple effects you'd expect from a button. Check the code from the first example in the vuetify [docs on menus](https://vuetifyjs.com/en/components/menus/)

Reply:
```vue
    <v-menu v-model="showGameMenu">
      <!-- Activator -->
      <template #activator="{ props }">
        <v-btn class="ml-0" id="game-menu-activator" v-bind="props" icon variant="text">
          <v-icon color="neutral-lighten-2" icon="mdi-cog" />
        </v-btn>
      </template>
      <!-- Menu -->
      <v-list id="game-menu" class="bg-surface-2">
        <v-list-item data-cy="rules-open" @click="openRulesDialog"> Rules </v-list-item>
        <!-- Concede Dialog (Initiate + Confirm) -->
        <v-list-item data-cy="concede-initiate" @click="openConcedeDialog"> Concede </v-list-item>
        <v-list-item data-cy="stalemate-initiate" @click="openStalemateDialog">
          Request Stalemate
        </v-list-item>
      </v-list>
    </v-menu>
``` 
I did it like that, i am not sure its ok, but it looks fine to me, thanks for the review !